File: java/src/test/java/com/agview/api/example/OAuthAccessToken.java
package com.agview.api.example;

import java.math.BigInteger;

public class OAuthAccessToken {
    private String access;
    private long exp;

    public String getAccess() {
        return access;
    }

    public void setAccess(String access) {
        this.access = access;
    }

    public long getExp() {
        return exp;
    }

    public void setExp(long exp) {
        this.exp = exp;
    }
}

File: c-sharp/Api/OAuthAccessToken.cs
namespace Npb.Agview.Api.Example
{
    public class OAuthAccessToken
    {
        public string Access { get; set; }
        public long Exp { get; set; }

        public override string ToString()
        {
            return $"Access Token = {Access}, Expiration Time (seconds) = {Exp}";
        }
    }
}